HW_5/myproject/hw2_shop_app/admin.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib import admin
+from .models import Product, Client, Order
+
+
+@admin.action(description="сбросить кол в ноль")
+def reset_quantity(modeladmin, request, queryset):
+    queryset.update(quantity=0)
+
+
+@admin.action(description="товар оплачен")
+def is_paid(modeladmin, request, queryset):
+    queryset.update(is_paid=True)
+
+
+class ProductAdmin(admin.ModelAdmin):
```

Comment:
Можно использовать декоратор 
```suggestion
@admin.register(Product)
class ProductAdmin(admin.ModelAdmin):
```
для регистрации кастомных моделей